.github/workflows/manage-neon-database.yml

```diff
@@ -0,0 +1,51 @@
+name: Manage Neon Database
+
+# This workflow triggers on every push to the "main or preprod" branch
+on:
+  push:
+    branches:
+      - preprod
+      - main
+
+jobs:
+  create-db:
```

Comment:
So this always create a new db everytime we push ?

Reply:
The job name has been changed to manage-db to more accurately describe what it does. Instead of creating a new database every time, it focuses on managing database operations based on the existing setup. The new name is simply meant to provide better clarity about its purpose.